net/netbird/Makefile

```
@@ -29,7 +29,8 @@ define Package/netbird
   SUBMENU:=VPN
   TITLE:=Connect your devices into a single secure private WireGuard®-based mesh network
   URL:=https://netbird.io
-  DEPENDS:=$(GO_ARCH_DEPENDS)
+  DEPENDS:=$(GO_ARCH_DEPENDS) \
+    +kmod-wireguard
```

```suggestion
  DEPENDS:=$(GO_ARCH_DEPENDS) +kmod-wireguard
```

it's unnecessary to wrap a new line for it. also please bump PKG_RELEASE.

---

Thanks for the review, I made the requested changes.